packages/cli/tsconfig.json

```diff
@@ -1,17 +1,14 @@
 {
+  "extends": "../../tsconfig",
   "compilerOptions": {
-    "rootDir": "src",
     "outDir": "build",
-    "declarationDir": "build/types",
-    "target": "es5",
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "declaration": true,
-    "sourceMap": true,
+    "lib": [
+      "es2015",
+      "es5",
+      "dom",
```

Comment:
A bit odd that we use dom globals in cli though.
Also we can probably add newer es libs

Reply:
Use it for WebAssembly and URL